src/confighandlerexception.cpp

```diff
@@ -17,13 +19,17 @@ const char* ConfigHandlerException::get_errmsg(ActionHandlerStatus status)
 		return _("invalid parameters.");
 	case ActionHandlerStatus::TOO_FEW_PARAMS:
 		return _("too few parameters.");
+	case ActionHandlerStatus::TOO_MANY_PARAMS:
+		return _("too many parameters.");
 	case ActionHandlerStatus::INVALID_COMMAND:
 		return _("unknown command (bug).");
 	case ActionHandlerStatus::FILENOTFOUND:
 		return _("file couldn't be opened.");
-	default:
-		return _("unknown error (bug).");
+	case ActionHandlerStatus::VALID:
+		assert(0 && "unreachable, because this function is only called on errors");
```

Comment:
It looks like the `VALID` option is never used, so I think it is better to just remove it from `configparser.h`.

Reply:
Right; thanks! Fixed in 7efb07a.